cdap-app-templates/cdap-etl/cdap-etl-tools/src/main/java/co/cask/cdap/etl/tool/UpgradeTool.java

```diff
@@ -481,7 +324,44 @@ private static boolean shouldUpgrade(ArtifactSummary artifactSummary) {
     // check its 3.2.x versions of etl
     ArtifactVersion artifactVersion = new ArtifactVersion(artifactSummary.getVersion());
     return !(artifactVersion.getMajor() == null || artifactVersion.getMinor() == null) &&
-      artifactVersion.getMajor() == 3 && artifactVersion.getMinor() == 2;
+      artifactVersion.getMajor() == 3 && (artifactVersion.getMinor() == 2 || artifactVersion.getMinor() == 3);
```

Comment:
should this be `(artifactVersion.getMinor() >= 2)` if we want to support other minor releases